.github/workflows/create-release.yml

```diff
@@ -0,0 +1,95 @@
+name: "Create release"
+
+on:
+  workflow_call:
```

Comment:
We should discourage users from triggering this directly, could you add a note that will show up in the GH UI to encourage users to trigger the `api` release instead?

Reply:
This is different from `workflow_dispatch`. This is to allow calling from another workflow.